fastapi/dependencies/utils.py

```diff
@@ -772,7 +772,7 @@ def request_params_to_args(
                 )
         value = _get_multidict_value(field, received_params, alias=alias)
         if value is not None:
-            params_to_process[field.name] = value
+            params_to_process[field.alias] = value
```

Comment:
Instead of changing this, we can also consider having both options: 

```suggestion
            params_to_process[field.name] = value
            params_to_process[field.alias] = value
```
This would still make the tests succeed, and it seems less risky in terms of breaking anything else. As I'm not super confident with this part of the code base, I'll ask @tiangolo for another review as well.

Reply:
The same mechanism is used when unpacking body fields. I think we need to make the same code.

https://github.com/fastapi/fastapi/blob/b6f6818d76f312fe30d798e52fff16927649db0a/fastapi/dependencies/utils.py#L867-L869

Reply:
I changed the tests and now I can say for sure that we cannot add a value to the `params_to_process` dictionary using `field.name`.